From bf63cbaa6b936d6cd6eb467d0225f3cef62a5b5d Mon Sep 17 00:00:00 2001
From: Huiquan Zhong <huiquan.zhong@intel.com>
Date: Wed, 15 Apr 2015 00:25:25 +0800
Subject: [PATCH] serial: HSU: modify some symbols which was not declared to
 static

Some symbols which was not declared, should modify to static.

Change-Id: Ic4a7e431c6547aa1c18e7bc9d304ff53120f28c5
Signed-off-by: Huiquan Zhong <huiquan.zhong@intel.com>
---
 drivers/tty/serial/hsu/hsu_plat.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/tty/serial/hsu/hsu_plat.c b/drivers/tty/serial/hsu/hsu_plat.c
index ab049f3..5956f62 100644
--- a/drivers/tty/serial/hsu/hsu_plat.c
+++ b/drivers/tty/serial/hsu/hsu_plat.c
@@ -46,7 +46,7 @@ static irqreturn_t wakeup_irq(int irq, void *data)
 	return IRQ_HANDLED;
 }
 
-int cht_hw_set_rts(struct uart_hsu_port *up, int value)
+static int cht_hw_set_rts(struct uart_hsu_port *up, int value)
 {
 	struct hsu_port_pin_cfg *pin_cfg = &up->port_cfg->pin_cfg;
 	struct gpio_desc *gpio;
@@ -77,7 +77,7 @@ int cht_hw_set_rts(struct uart_hsu_port *up, int value)
 	return 0;
 }
 
-int cht_hsu_hw_suspend(struct uart_hsu_port *up)
+static int cht_hsu_hw_suspend(struct uart_hsu_port *up)
 {
 	struct hsu_port_pin_cfg *pin_cfg = &up->port_cfg->pin_cfg;
 	struct gpio_desc *gpio;
@@ -126,7 +126,7 @@ int cht_hsu_hw_suspend(struct uart_hsu_port *up)
 	return 0;
 }
 
-int cht_hsu_hw_resume(struct uart_hsu_port *up)
+static int cht_hsu_hw_resume(struct uart_hsu_port *up)
 {
 	struct hsu_port_pin_cfg *pin_cfg = &up->port_cfg->pin_cfg;
 
@@ -156,7 +156,7 @@ int cht_hsu_hw_resume(struct uart_hsu_port *up)
 	return 0;
 }
 
-void cht_hsu_reset(void __iomem *addr)
+static void cht_hsu_reset(void __iomem *addr)
 {
 	writel(0, addr + CHT_HSU_RESET);
 	writel(3, addr + CHT_HSU_RESET);
@@ -165,7 +165,7 @@ void cht_hsu_reset(void __iomem *addr)
 	writel(2, addr + CHT_HSU_OVF_IRQ);
 }
 
-void cht_hsu_set_clk(unsigned int m, unsigned int n,
+static void cht_hsu_set_clk(unsigned int m, unsigned int n,
 				void __iomem *addr)
 {
 	unsigned int param, update_bit;
@@ -203,7 +203,7 @@ static void hsu_serial_setup(struct uart_hsu_port *up)
 	p->set_termios = hsu_set_termios;
 }
 
-unsigned int cht_hsu_get_uartclk(struct uart_hsu_port *up)
+static unsigned int cht_hsu_get_uartclk(struct uart_hsu_port *up)
 {
 	struct clk *clk;
 	unsigned int uartclk = 0;
-- 
1.9.1

